### PR TITLE
best practices for FRC: API cyber security

### PR DESCRIPTION
**Pull request by abinabra**
Adding new documents for food remedy co cyber security.

### CODE REVIEW

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+- **Access Controls:** Assessing the effectiveness of access controls and permissions to prevent unauthorized access to sensitive data or systems.
+- **Data Protection:** Examining data storage, encryption, and handling practices to safeguard sensitive information.
+- **Network Security:** Evaluating firewalls, intrusion detection/prevention systems, and network segmentation to prevent unauthorized access.
+- **Application Security:** Assessing the security of applications, including code quality, vulnerability assessments, and secure coding practices.
+- **Configuration Management:** Reviewing system and network configurations to ensure they align with security best practices.
+- **Physical Security:** Reviewing physical access controls to data centers and other critical facilities.
+- **Incident Response:** Evaluating the organization's incident response plan and its readiness to address security incidents.
+- **Compliance:** Ensuring compliance with industry regulations and legal requirements.
+
+## Accidental Exposure using OSINT (Open Source Intelligence)
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:23]: Heading are confusing, better to use different word size for heading, sub-heading and the information.

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+- **Wildcard Operator:** Use the * wildcard to capture variations of a term. For instance, "API key=*" will search for different API key values.
+- **Intitle Operator:** Use the intitle: operator to find pages with specific terms in their title. For example, intitle:"login credentials".
+- **Boolean Operators:** Combine operators like AND, OR, and NOT to refine your search. For example, password OR "API key" NOT tutorial.
+- **Search Specific Domains:** Target specific domains that might be relevant to your search, like specific websites or forums where developers discuss code.
+
+**Example:** "password=12345" OR "API key=abcdef" -site:github.com -site:stackoverflow.com filetype:xml
+
+Additionally, you might consider using dedicated tools and platforms designed for searching for leaked or exposed credentials on the internet, as they can provide more focused and comprehensive results. Examples of such tools include Shodan, Censys, and Have I Been Pwned.
+
+### Points to Keep in Mind
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:66]: Formatting issue, consider a tab space for the sub-point and clear the extra line in between the Main point and sub-points.

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+
+- Scenario: Failing to set up proper logging and monitoring for IAM actions.
+- Risk: Difficulty in detecting unauthorized or suspicious activities, hindering incident response.
+
+### 5. Unused Credentials and Access Keys:
+
+- Scenario: Leaving unused IAM users, roles, or access keys active.
+- Risk: Increased attack surface, potential for unauthorized access, and resource wastage.
+
+### Managing IAM Policies and Security Best Practices
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:114]: Formatting issue, consider a tab space for the sub-point and clear the extra line in between the Main point and sub-points.

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 4]
```diff
@@ -0,0 +1,451 @@
...
+
+- Use Customer-Supplied Encryption Keys (CSEK) for more control over encryption keys.
+- Enable encryption in transit using HTTPS and SSL/TLS.
+
+6. **Compliance and Auditing**
+
+- Leverage Google Cloud Compliance programs for specific industry standards.
+- Use Cloud Asset Inventory for compliance auditing.
+
+### Microsoft Azure
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:367]: Formatting issue, consider a tab space for the sub-point and clear the extra line in between the Main point and sub-points.

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 5]
```diff
@@ -0,0 +1,451 @@
...
+
+- Enable encryption at rest using Azure Key Vault.
+- Implement Transport Layer Security (TLS) for data in transit.
+
+6. **Compliance and Auditing**
+
+- Utilize Azure Policy and Azure Blueprints for compliance enforcement.
+- Leverage Azure Security Center's compliance assessment capabilities.
+
+### Amazon Web Services (AWS)
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:399]: Formatting issue, consider a tab space for the sub-point and clear the extra line in between the Main point and sub-points.

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 6]
```diff
@@ -0,0 +1,451 @@
...
+
+- Practice: Enable MFA for IAM users to add an additional layer of security.
+- Benefit: Reduces the risk of unauthorized access even if credentials are compromised.
+
+5. **Centralized Access Control:**
+
+- Practice: Use AWS Identity and Access Management (IAM) to centrally manage user access.
+- Benefit: Provides a unified way to manage permissions across AWS services.
+
+## Inculcating Good Security Practices into the Project
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:141]: Formatting issue, consider a tab space for the sub-point and clear the extra line in between the Main point and sub-points.

File: food-remedy-co/frc-api-cyber-security-best-practices.md [hunk 7]
```diff
@@ -0,0 +1,451 @@
...
+- Integrate security bots or plugins to monitor and prevent sensitive data sharing.
+- Conduct regular security training sessions for board members.
+
+### 7. Conclusion
+
+Conducting a security audit of a Trello account is crucial to maintaining a secure environment for collaboration and data sharing. By following this comprehensive guide, you can identify vulnerabilities, mitigate risks, and implement best practices to enhance the overall security posture of your Trello account.
+
+## Cloud Security Posture Assessment
+
+### Google Cloud Platform (GCP)
```
Comment (SamrajMahadev) [inline food-remedy-co/frc-api-cyber-security-best-practices.md:335]: Formatting issue, consider a tab space for the sub-point and clear the extra line in between the Main point and sub-points.

File: food-remedy-co/frc-api-cyber-security-best-practices.md (hunk 1 above)
Comment (abinabra) [reply food-remedy-co/frc-api-cyber-security-best-practices.md:23]: Thanks for the review and I will make the changes

File: food-remedy-co/frc-api-container-hardening.md
Comment (deasta) [inline food-remedy-co/frc-api-container-hardening.md:1]: Fix headings, combine references and make sure to share with relevant teams through Trello and Teams.

**Pull request merged**